libs/gpio/pin_defs.h

```diff
@@ -0,0 +1,88 @@
+#include <avr/io.h>
```

Comment:
Does this imply an order in which `avr.io.h` and `gpio/api.h` should be included, if including both at the application layer? If so, is that documented somewhere? I could see that being a big "gotcha" down the road, but I'm also not confident this is actually even an issue lol